python/spglib/spglib.py

```diff
@@ -640,6 +642,7 @@ def get_symmetry(
         warnings.warn(
             "Use get_magnetic_symmetry() for cell with magnetic moments.",
             DeprecationWarning,
+            stacklevel=2,
```

Comment:
This one should be stacklevel=1

Reply:
Changed. But can I ask you why?

Reply:
I'm curious why using one stack here too, IMO `stacklevel=1` is useful when you want to point to the implementation itself instead of the API. But the deprecation is apparently pointing to the `get_symmetry` interface?

Reply:
Because the usage of `get_symmetry` is not deprecated, it's just one mode of operation is deprecated. When we change to `warnings.deprecated` we can move it to the typing part and distinguish between those.

Thinks about it more I think `stacklevel=2` could also work in some sense, but I would want it to show up as any usage of `get_symmetry` is deprecated. Probably we can change it once the mypy part is also added.